### PR TITLE
Refs [Task] [34300] Create UI search screen

### PR DESCRIPTION
**Pull request by hunghuytra28081**
## Related Tickets
- [#Task 34300: Create UI search screen](https://edu-redmine.sun-asterisk.vn/issues/34300)
## WHAT
- Create UI search screen
## Evidence 
image
## Review Checklist

Category | View Point | Description | Expected Reviewer Answer | Self review | Reviewer2 (name)
--- | --- | --- | --- | --- | ---
Conventions | Does the code follow Sun* coding style and coding conventions? | https://github.com/framgia/coding-standards/tree/master/eng/android | YES |<li>- [ ] yes</li>|<li>- [ ] yes</li>  
Redmine | Does the ticket follow Sun* Redmine working process?  | https://github.com/framgia/Training-Guideline/blob/master/WorkingProcess/redmine/redmine.md| YES |<li>- [ ] yes</li>|<li>- [ ] yes</li>  
Documentation | Is there any incomplete code? If so, should it be removed or flagged with a suitable marker like ‘TODO’? |  | YES |<li>- [ ] yes</li>|<li>- [ ] yes</li>  
## Notes (Optional)

### CODE REVIEW

File: app/src/main/java/com/sun/imagegif/ui/main/page/PageContainerFragment.kt [hunk 1]
```diff
@@ -10,7 +10,7 @@ import com.sun.imagegif.R
 import com.sun.imagegif.ui.home.HomeFragment
 import com.sun.imagegif.ui.search.SearchFragment
 import com.sun.imagegif.ui.storage.StorageFragment
-import com.sun.imagegif.ui.utils.MenuItem
+import com.sun.imagegif.utils.MenuItem
```
Comment (tuanlvt-1117) [inline app/src/main/java/com/sun/imagegif/ui/main/page/PageContainerFragment.kt:13]: remove

File: app/src/main/java/com/sun/imagegif/ui/search/SearchContact.kt [hunk 2]
```diff
@@ -0,0 +1,10 @@
+package com.sun.imagegif.ui.search
+
+import com.sun.imagegif.utils.BasePresenter
+
+interface SearchContact {
```
Comment (tuanlvt-1117) [inline app/src/main/java/com/sun/imagegif/ui/search/SearchContact.kt]:
```suggestion
interface SearchContract {
```

File: app/src/main/java/com/sun/imagegif/ui/search/SearchFragment.kt [hunk 3]
```diff
@@ -17,6 +18,11 @@ class SearchFragment : Fragment() {
         return inflater.inflate(R.layout.fragment_search, container, false)
     }
 
+    override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
+        super.onViewCreated(view, savedInstanceState)
+        tabLayoutSearch.addTab(tabLayoutSearch.newTab().setText("GIF"))
+        tabLayoutSearch.addTab(tabLayoutSearch.newTab().setText("Text"))
```
Comment (tuanlvt-1117) [inline app/src/main/java/com/sun/imagegif/ui/search/SearchFragment.kt]: scope function

File: app/src/main/java/com/sun/imagegif/ui/search/SearchPresenter.kt [hunk 4]
```diff
@@ -0,0 +1,14 @@
+package com.sun.imagegif.ui.search
+
+class SearchPresenter : SearchContact.Presenter {
+
+    private var view : SearchContact.View? = null
+
+    override fun onStart() = Unit
+
+    override fun onStop() = Unit
```
Comment (tuanlvt-1117) [inline app/src/main/java/com/sun/imagegif/ui/search/SearchPresenter.kt:9]: sao lại có return Unit ở đây e?

File: app/src/main/res/layout/fragment_search.xml [hunk 5]
```diff
@@ -1,6 +1,57 @@
...
+        app:layout_constraintHeight_percent="0.095"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toTopOf="parent">
+
+        <androidx.appcompat.widget.SearchView
+            android:id="@+id/searchBar"
+            android:layout_width="0dp"
+            android:layout_height="0dp"
+            android:layout_marginStart="@dimen/dp_55"
+            android:layout_marginEnd="@dimen/dp_55"
```
Comment (tuanlvt-1117) [inline app/src/main/res/layout/fragment_search.xml:25]: sao lại margin lớn vậy e?

File: app/src/main/java/com/sun/imagegif/ui/search/SearchPresenter.kt (hunk 4 above)
Comment (hunghuytra28081) [reply app/src/main/java/com/sun/imagegif/ui/search/SearchPresenter.kt:9]: dạ cái này em chưa dùng nên em cho v anh, thay cho cặp {}

File: app/src/main/res/layout/fragment_search.xml (hunk 5 above)
Comment (hunghuytra28081) [reply app/src/main/res/layout/fragment_search.xml:25]: dạ thanh search anh, nó magin với Constrain nhỏ trong cái Constrain lớn

File: app/src/main/java/com/sun/imagegif/ui/main/page/PageContainerFragment.kt (hunk 1 above)
Comment (hunghuytra28081) [reply app/src/main/java/com/sun/imagegif/ui/main/page/PageContainerFragment.kt:13]: dạ này em em chuyển từ thư mục utils trong package ui vào chung package utils cho gọn anh

File: app/src/main/java/com/sun/imagegif/ui/search/SearchFragment.kt [hunk 6]
```diff
@@ -17,6 +18,14 @@ class SearchFragment : Fragment() {
         return inflater.inflate(R.layout.fragment_search, container, false)
     }
 
+    override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
+        super.onViewCreated(view, savedInstanceState)
+        tabLayoutSearch.apply {
+            addTab(tabLayoutSearch.newTab().setText("GIF"))
+            addTab(tabLayoutSearch.newTab().setText("Text"))
```
Comment (tuanlvt-1117) [inline app/src/main/java/com/sun/imagegif/ui/search/SearchFragment.kt]: GIF, Text
hard code.

**Review by tuanlvt-1117 — APPROVED** [review]

**Pull request merged**